actors/verifreg/src/lib.rs

```diff
@@ -328,12 +335,18 @@ impl Actor {
                 }
 
                 for id in to_remove {
-                    let existing = allocs.remove(params.client, *id).context_code(
-                        ExitCode::USR_ILLEGAL_STATE,
-                        format!("failed to remove allocation {}", id),
-                    )?;
+                    let existing = allocs
+                        .remove(params.client, *id)
+                        .context_code(
+                            ExitCode::USR_ILLEGAL_STATE,
+                            format!("failed to remove allocation {}", id),
+                        )?
+                        .unwrap();
+
+                    emit::allocation_removed(rt, *id)?;
+
                     // Unwrapping here as both paths to here should ensure the allocation exists.
```

Comment:
Please move this comment up  next to the unwrap().

Reply:
Done.